src/OrchardCore.Modules/OrchardCore.Localization/Extensions/LocalizationApplicationBuilderExtensions.cs

```diff
@@ -1,21 +0,0 @@
-using System;
-using Microsoft.Extensions.DependencyInjection;
-using OrchardCore.Localization;
-using OrchardCore.Localization.Services;
-
-namespace Microsoft.AspNetCore.Builder
-{
-    public static class LocalizationApplicationBuilderExtensions
-    {
-        public static IApplicationBuilder MapCulturesAlias(this IApplicationBuilder app)
```

Comment:
Okay to use an extension helper when we have special things to do, but here we don't need to resolve ourself `ICultureAliasProvider`, we can just let aspnet do it when it activates the middleware, then why using an extension to just do `app.UseMiddleware<CultureAliasMiddleware>()`.

I will look at this PR again this night because I asked to me other questions.

For example here when we map a Culture, for now we don't take into account the current `IgnoreSystemSettings` options.

Reply:
> For example here when we map a Culture, for now we don't take into account the current IgnoreSystemSettings options.

Let me check this case ..